sdk/python/tests/expediagroup/test_milvus_online_store.py

```diff
@@ -431,3 +455,81 @@ def test_milvus_online_write_batch(self, repo_config, caplog):
             collection = Collection(name=self.collection_to_write)
             progress = utility.index_building_progress(collection_name=collection.name)
             assert progress["total_rows"] == total_rows_to_write
+
+    def test_milvus_online_read(self, repo_config, caplog):
+
+        # Generating data to directly load into Milvus
+        data_to_load = [
+            [i for i in range(10)],
+            [i + 2000 for i in range(10)],
+            [[random.random() for _ in range(2)] for _ in range(10)],
+        ]
+
+        # Creating a common schema for collection to directly add to Milvus
+        schema = CollectionSchema(
+            [
+                FieldSchema("film_id", DataType.INT64, is_primary=True),
+                FieldSchema("film_date", DataType.INT64),
+                FieldSchema("films", dtype=DataType.FLOAT_VECTOR, dim=2),
+            ]
+        )
+
+        # Creating an end-to-end example with just using pymilvus to understand how it works
+        with MilvusConnectionManager(repo_config.online_store):
+            # Create a collection
+            collection = Collection(name=self.collection_to_write, schema=schema)
+            # Drop all indexes if any exists
+            collection.drop_index()
+            # Create a new index
+            index_param = {"index_type": "FLAT", "metric_type": "L2", "params": {}}
+            collection.create_index("films", index_param)
+            collection.insert(data_to_load)
+            collection.load()
+
+        feast_schema = [
+            Field(
+                name="film_id",
+                dtype=Int64,
+                tags={"is_primary": "True", "description": "int64"},
```

Comment:
is_primary tags are obsolete and should be removed

Reply:
What do you mean? How is primary key defined?

Reply:
the entity is the primary key